tests/src/drivers/test_driver_key_management.c

```diff
@@ -229,15 +232,15 @@ psa_status_t mbedtls_test_opaque_export_key(
     const uint8_t *key, size_t key_length,
     uint8_t *data, size_t data_size, size_t *data_length )
 {
+    /* Assume this is a builtin key based on the key material length. */
```

Comment:
This was not changed in 5282 thus no point in changing it in this backport I'd say.

Reply:
The structure of the `mbedtls_test_opaque_export_key` is different in the `development`, and the variable is declared there in a smaller scope, so I would rather consider this fix applicable to 2.28 only:

```
    if( key_length == sizeof( psa_drv_slot_number_t ) )
    {
        /* Assume this is a builtin key based on the key material length. */
        psa_drv_slot_number_t slot_number = *( ( psa_drv_slot_number_t* ) key );

        switch( slot_number )
        {
			...
        }
    }
    else
    {
        ...
    }
    return( PSA_ERROR_NOT_SUPPORTED );
```
If you still want me to remove it, I'll do it, but I think that it's a good improvement.

Reply:
ok, this fits well in commit "Test driver: keep variable declarations first " thus ok to keep it.